packages/web-components/src/components/combo-button/combo-button.stories.ts

```diff
@@ -0,0 +1,252 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit';
+import './index';
+import CopyFile16 from '@carbon/icons/lib/copy--file/16';
+import Export16 from '@carbon/icons/lib/export/16';
+
+const args = {
+  label: 'Primary action',
+  size: 'lg',
+  menuAlignment: 'bottom',
+};
+
+const argTypes = {
+  disabled: {
+    control: 'boolean',
+    description: 'Specify whether the ComboButton should be disabled, or not.',
+  },
+  label: {
+    control: 'text',
+    description: 'Provide the label to be rendered on the trigger button.',
+  },
+  menuAlignment: {
+    control: 'select',
+    description:
+      'Experimental property. Specify how the menu should align with the button element',
+    options: [
+      'top',
+      'top-start',
+      'top-end',
+      'bottom',
+      'bottom-start',
+      'bottom-end',
+    ],
+  },
+  onClick: {
+    control: true,
+    description:
+      'Provide an optional function to be called when the primary action element is clicked.',
+  },
+  size: {
+    control: 'radio',
+    description: `Specify the size of the button and menu.
+'sm'
+'md'
+'lg'`,
+    options: ['sm', 'md', 'lg'],
+  },
+  tooltipAlignment: {
+    control: 'radio',
+    description: 'Specify how the trigger tooltip should be aligned.',
+  },
+};
+
+export const Default = {
+  argTypes: argTypes,
+  args: args,
+  render: ({
+    disabled,
+    label,
+    menuAlignment,
+    size,
+    tooltipAlignment,
+    onClick,
+  }) => html`
+    <cds-combo-button
+      label="${label}"
+      menu-alignment=${menuAlignment}
+      size=${size}
+      ?disabled=${disabled}
+      .tooltip-alignment="${tooltipAlignment}"
+      .onClick=${onClick}>
+      <cds-menu>
+        <cds-menu-item
+          label="Second action with a long label description"
+          .handleClick=${onClick}></cds-menu-item>
```

Comment:
We shouldn't need to have the author set the `handleClick` for each menu item. The `onClick` on the combo-button should handle the click event for them.
```suggestion
          ></cds-menu-item>
```